test/testmessage.c

```diff
@@ -213,6 +213,15 @@ int main(int argc, char *argv[])
         quit(1);
     }
     
+    success = SDL_ShowSimpleMessageBox(SDL_MESSAGEBOX_ERROR,
+                                       "Arabic (multi line)",
+                                       "سطر طويل جدًا من النص\nخط قصير\nسطر طويل للغاية من النص مذهل بشكل لا يصدق في اللغة العربية التي يتم التحدث بها في منطقة الشرق الأوسط وشمال أفريقيا",
```

Comment:
How does this behave on other video backends? Wayland, Windows and/or macOS?
(I think it's fine if it's unknown/untested: the api surface is unchanged)